lib/compress.js

```diff
@@ -3028,63 +3028,49 @@ merge(Compressor.prototype, {
         return this;
     });
 
-    var commutativeOperators = makePredicate("== === != !== * & | ^");
+    var reversibleOperators = makePredicate("+ * & | ^ < > <= >= == != === !==");
```

Comment:
`+` is not reversible if left or right is a string.

Inequality operators should not be here.